components/package-template/src/sbin/.common-env.sh

```
@@ -0,0 +1,53 @@
+    version="$(tr -d '[:space:]' < "$version_file")"
+    export CLP_PACKAGE_CONTAINER_IMAGE_REF="ghcr.io/y-scope/clp/clp-package:v$version"
+else
+    echo "Error: Neither clp-package-image.id nor VERSION file exists." >&2
+    return 1 2>/dev/null || exit 1
+fi
+
+uid="$(id --user 2>/dev/null || echo "1000")"
+gid="$(getent group docker | cut -d: -f3 2>/dev/null || echo "999")"
+export CLP_FIRST_PARTY_SERVICE_UID_GID="$uid:$gid"
+
+export CLP_PWD_HOST="$(pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)